### ISSUE TITLE
zCom4 & zText4 modules?

### ISSUE THREAD

**Issue by DavidHaslam** [JohnAustinDev/xulsword#244]
Support for **zCom4** and **zText4** never made it to SWORD 1.7.4 and will require SWORD 1.8 or later.

However, **JSword** has just been enhanced to support these `ModDrv` values.

The .conf file for such a module should include: `MinimumVersion=1.8`

What might happen to **xulsword** if it encounters a **zText4** module?
i.e. Will it crash or otherwise behave unpredicably?
I imagine it should simply drop the module when the program opens.

**Comment by JohnAustinDev** [comment]
xulsword has for a while been built from SWORD svn revision 3203 which supports zText4. So zText4 modules aren't a problem for xulsword's engine. However zText4 is an unexpected module type for xulsword's module installer, so they are being rejected by version 3.14. But xulsword 3.15's installer will recognize and allow them.

**Comment by DavidHaslam** [comment]
I suppose the symptom would be that such a module would be recorded as **Dropping** in the `consoleLog.txt` file.  As indeed it was for the **RusCARS4** module.

```
ModuleDirectory="C:\Users\David\AppData\Roaming\IBT\MK\Profiles\resources"
ERROR: Dropping module "FreBDM1707". enginevers:1.7.3 < xsengvers:1.8.0
ERROR: Dropping module "KorRV". enginevers:1.7.3 < xsengvers:1.7.9
ERROR: Dropping module "RusCARS4". enginevers:1.7.3 < xsengvers:1.8
ERROR: Dropping module "deu1912eb". enginevers:1.7.3 < xsengvers:1.7.4
ERROR: Dropping module "engAMP1987eb". enginevers:1.7.3 < xsengvers:1.7.4
ERROR: Dropping module "engHCSB2009eb". enginevers:1.7.3 < xsengvers:1.7.4
ERROR: Dropping module "engNASB1995eb". enginevers:1.7.3 < xsengvers:1.7.4
ERROR: Dropping module "engNET2006eb". enginevers:1.7.3 < xsengvers:1.7.4
ERROR: Dropping module "engNIV2011eb". enginevers:1.7.3 < xsengvers:1.7.4
ERROR: Dropping module "engerv2006eb". enginevers:1.7.3 < xsengvers:1.7.4
ERROR: Dropping module "fraLSG1910eb". enginevers:1.7.3 < xsengvers:1.7.4
ERROR: Dropping module "grcTischeb". enginevers:1.7.3 < xsengvers:1.7.4
ERROR: Dropping module "hboMTeb". enginevers:1.7.3 < xsengvers:1.7.4
ERROR: Dropping module "hboWLCeb". enginevers:1.7.3 < xsengvers:1.7.4
ERROR: Dropping module "pon1922eb". enginevers:1.7.3 < xsengvers:1.7.4
ERROR: Dropping module "ponPDNeb". enginevers:1.7.3 < xsengvers:1.7.4
ERROR: Dropping module "thaKJV2003eb". enginevers:1.7.3 < xsengvers:1.7.4
```

The complete file is attached (FIO).

[consoleLog.txt](https://github.com/JohnAustinDev/xulsword/files/758417/consoleLog.txt)

**Comment by DavidHaslam** [comment]
Module **RusCARS4** was rebuilt from the OSIS XML file by myself in order to see how other front-ends fare.
Same content as module **RusCARS** but using the bigger record size.

btw. The module names ending with "eb" are from the `eBible.org` repository.
The latest SWORD release is 1.7.4 so I don't know why xulsword 3.14 was still compiled against 1.7.3
See https://www.crosswire.org/wiki/Sword_library_versions

**Comment by DavidHaslam** [comment]
#### Aside:

Module **FreBDM1707** is the **French Bible David Martin 1707** that was built using one of the new versification schemes for French translations prepared by **DomCox**. 

These **av11n**s will only become available in SWORD 1.8 or later.

The module was also made by **DomCox** and sent to modules at crosswire.

**Comment by DavidHaslam** [comment]
#### Aside:

I've really no idea why module **KorRV** has 
```
MinimumVersion=1.7.9
SwordVersionDate=2013-10-14
```

Methinks this must be a typo in the conf file, as it should probably be OK with 
`MinimumVersion=1.5.9`
